Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -484,8 +484,11 @@ Ref<WebPage> WebPage::create(PageIdentifier pageID, WebPageCreationParameters&&
         WebProcess::singleton().injectedBundle()->didCreatePage(page.ptr());
 
 #if HAVE(SANDBOX_STATE_FLAGS)
-    auto auditToken = WebProcess::singleton().auditTokenForSelf();
-    sandbox_enable_state_flag("WebContentProcessLaunched", *auditToken);
+    static dispatch_once_t onceToken;
```

Comment:
We should use a global static boolean. We don't need a dispatch_once because we don't need locking because there is only a single thread involved.

Reply:
Ah, good point, will fix!

Thanks for reviewing!